pageserver/src/tenant/mgr.rs

```diff
@@ -206,20 +256,48 @@ pub async fn init_tenant_mgr(
                         }
                     };
 
+                    // Try loading the location configuration
+                    let mut location_conf = match Tenant::load_tenant_config(conf, &tenant_id) {
+                        Ok(c) => c,
+                        Err(e) => {
+                            warn!("Error loading tenant config, skipping the tenant ({e})");
```

Comment:
https://github.com/neondatabase/neon/pull/5178/files#diff-64e77fa41a0849414fe2490e400ce595f0dfc6b340bde62e11334417a0aac9fcR210-R217

also please +1 that PR

Reply:
updated in 9fcea2dbd